includes/class-wc-payments-order-service.php

```diff
@@ -216,7 +216,7 @@ public function mark_payment_capture_failed( $order, $intent_id, $intent_status,
 	}
 
 	/**
-	 * Updates an order to cancelled status, while adding a note with a link to the transaction.
+	 * Update an order to failed status, and add note with a link to the transaction. Context - when a Payment Intent expires.
```

Comment:
I suggest mentioning that failed statutes enable the buyer to attempt a new payment.

Reply:
Thanks. I have added a comment now 230c22f